pkg/storage/caching/caching.go

```diff
@@ -46,6 +48,16 @@ func (c *cachedOpenFGADatastore) ReadAuthorizationModel(ctx context.Context, sto
 	return model, nil
 }
 
+func (c *cachedOpenFGADatastore) FindLatestAuthorizationModelID(ctx context.Context, storeID string) (string, error) {
+	v, err, _ := c.G.Do(fmt.Sprintf("FindLatestAuthorizationModelID:%s", storeID), func() (interface{}, error) {
+		return c.OpenFGADatastore.FindLatestAuthorizationModelID(ctx, storeID)
```

Comment:
Can we please add a test for this to make sure that the mocked call to this method is not called more than once for 2 concurrent lookups?